Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -75,6 +76,29 @@ SoftwareGLContext::SoftwareGLContext(Gfx::Bitmap& frontbuffer)
     light0.diffuse_intensity = { 1.0f, 1.0f, 1.0f, 1.0f };
     light0.specular_intensity = { 1.0f, 1.0f, 1.0f, 1.0f };
     m_light_state_is_dirty = true;
+
+    m_client_side_texture_coord_array_enabled.resize(m_device_info.num_texture_units);
+    m_client_tex_coord_pointer.resize(m_device_info.num_texture_units);
+    m_current_vertex_tex_coord.resize(m_device_info.num_texture_units);
```

Comment:
Nitpick: semantically `AK::FixedArray` might be the better suited option here instead of `Vector` since we know the size of these arrays is fixed at runtime.

Reply:
True. I'm not sure how to use FixedArray. AFAICT the size has to be set upon construction time. So we'd have to initialize the FixedArray in SoftwareGLContext's member initializer list?

Reply:
Yes exactly, instead of the `.resize`s

Reply:
Oops merged already. I think this will make more sense if we transform context creation into the `try_create` pattern.